.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Bug report
+about: Create a report to help us improve the library
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Before raising, please check if somebody else has already reported your issue. -->
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### Steps to reproduce
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Environment
+* Bugsnag version:
+* Ruby version:
+* Bundle version:
+* Integration framework version:
+    * Que:
+    * Rack
+    * Rails:
+    * Rake:
+    * Sidekiq:
```

Comment:
Are these just the integrations we have specific docs for? It might be worth adding the other integrations we have too, but they could also come under "other" (I'm happy either way, just raising the question!)

e.g.

```suggestion
    * DelayedJob
    * Mailman
    * Mongo
    * Que:
    * Rack
    * Rails:
    * Rake:
    * Resque:
    * Shoryuken:
    * Sidekiq:
```

Reply:
Yes, I took the list based on the docs https://docs.bugsnag.com/platforms/ruby/.  In this case personally I'd prefer to match the docs by implying the rest under 'others'.